tools/simde_update.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# Copyright 2020 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+"""Updates the arm_neon.h header taken from SIMDe
+(https://github.com/simd-everywhere/simde) in system/include/neon
+"""
+
+
+import os
+import subprocess
+import sys
+
+from os import path
+
+# Why are the packages being stupid
+# from .shared import get_emscripten_temp_dir
+
+
+tmpdir = "/tmp"
+emdir = path.join(path.dirname(path.realpath(__file__)), "..")
+
+def main(simde_path=None):
+  if len(sys.argv) == 2:
+    simde_dir = sys.argv[1]
+  elif len(sys.argv) == 1:
+    simde_dir = None
+  else:
+    print('''USAGE:
+./simde_update.py [SIMDE_REPO_DIRECTORY]''')
+
+  if not simde_dir:
+    try:
+      os.mkdir(path.join(tmpdir, "simde"))
+      os.system("git clone git@github.com:simd-everywhere/simde " + path.join(tmpdir, "simde"))
+    except FileExistsError as e:
+      if not path.isdir(path.join(tmpdir, "simde")):
+        print("/tmp/simde not a directory, exiting...")
+        return 1
+      else:
+        print("simde repository already found in tmpdir, using found repository")
+        os.system("git -C " + path.join(tmpdir, "simde") + " pull")
```

Comment:
Will the error message be reasonable if these git commands fail?

Reply:
I added some more code there, let me know what you think of it.

Reply:
Actually, I decided on removing the code I added. With the emscripten temp dir there won't be any conflicts.

Reply:
Seems reasonable, especially since this will only be run by Emscripten devs.